code/modules/reagents/reagent_dispenser.dm

```diff
@@ -76,6 +76,13 @@
 		..()
 		reagents.add_reagent("water",1000)
 
+/obj/structure/reagent_dispensers/watertank/high/New()
+	name = "high-capacity water tank"

```

Comment:
It doesn't seem to actually be high-capacity, is it? Would it only be high-pressure? If the capacity is in fact larger and you can in fact empty it and refill it to 4000 units rather than 1000, please do correct me.

Reply:
Water tanks are just big reagent containers. It's seems that you're thinking about air tanks.

Reply:
I'm aware- the default allow you to fill it to 1000 unit and the desc states this one is high-pressure. The name should probably be high-pressure unless you can also empty and refill it to 4k units instead of 1k.

Reply:
You can't refill water tanks.

Reply:
Huh. I might be confusing it with another codebase, then.